sos/report/plugins/kubernetes.py

```diff
@@ -34,26 +67,14 @@ class Kubernetes(Plugin):
     ]
 
     kube_cmd = "kubectl"
-    resources = [
-        'deployments',
-        'ingresses',
-        'limitranges',
-        'pods',
-        'policies',
-        'pvc',
-        'rc',
-        'resourcequotas',
-        'routes',
-        'services'
-    ]
 
     def check_is_master(self):
         """ Check if this is the master node """
         return any(self.path_exists(f) for f in self.files)
 
     def setup(self):
-        self.add_copy_spec("/etc/kubernetes")
-        self.add_copy_spec("/run/flannel")
+        self.plugin_timeout = 1200
```

Comment:
`plugin_timeout` needs to be in the class definition, alongside `packages` or `services`, etc...